Makefile

```diff
@@ -180,6 +202,8 @@ validate-repo-no-changes:
 %.pb.go: %.proto
 	$(Q)echo "Generating $@..."; \
 	$(PROTO_COMPILE) $<
+	sed -i 's;\(//.*protoc.*\)v.*;\1[stripped];g' $(shell find . -name '*.pb.go')
+	sed -i '1s;^;//go:build !tinygo.wasm\n\n;' pkg/api/api_ttrpc.pb.go
```

Comment:
@saschagrunert I see why the tinygo.wasm exclusion is necessary, but what about the second one which strips the versions ? Is the idea there to filter out 'changes' resulting from folks using different toolchain versions, as long as those versions generate identical output (apart from the version comment tag) ?

Reply:
I removed that part, yes it was about version inconsistencies.